### PR TITLE
Adjusting a layers opacity with a slider example

### PR DESCRIPTION
**Pull request by tristen**
layer-opacity-example

cc @jfirebaugh for review

### CODE REVIEW

**Comment by lucaswoj** [comment]
Changing the opacity of the water feels contrived. Can we add a data overlay and change its opacity?

File: docs/_posts/examples/3400-01-23-adjust-layer-opacity.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: example
+category: example
+title: Adjust a layers opacity
```
Comment (jfirebaugh) [inline docs/_posts/examples/3400-01-23-adjust-layer-opacity.html]: "layer's"

**Comment by jfirebaugh** [comment]
Code is :+1:. I second a data layer (or image layer like https://www.mapbox.com/mapbox.js/example/v1.0.0/opacity/).

File: docs/_posts/examples/3400-01-23-adjust-layer-opacity.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+layout: example
+category: example
+title: "Adjust a layer's opacity"
+description: "Drag the range slider to adjust the opacity of the water layer in Mapbox streets."
```
Comment (jfirebaugh) [inline docs/_posts/examples/3400-01-23-adjust-layer-opacity.html]: Can you update the description too? It's no longer the water layer being adjusted.

**Pull request merged**